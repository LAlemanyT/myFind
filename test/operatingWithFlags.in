./myfind.py /home/a "ls {} -l" 
