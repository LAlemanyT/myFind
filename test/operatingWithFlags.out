total 8
drwxr-xr-x 3 user user 4096 Oct  6 04:52 b
-rw-r--r-- 1 user user   10 Oct  6 04:52 file.txt
-rw-r--r-- 1 user user 10 Oct  6 04:52 /home/a/file.txt
total 8
drwxr-xr-x 3 user user 4096 Oct  6 04:52 c
-rw-r--r-- 1 user user   10 Oct  6 04:52 file.txt
-rw-r--r-- 1 user user 10 Oct  6 04:52 /home/a/b/file.txt
total 4
drwxr-xr-x 2 user user 4096 Oct  6 04:52 file.txt
total 0
